go/vt/vttablet/tabletmanager/rpc_vreplication_test.go

```diff
@@ -323,11 +316,6 @@ func TestMoveTables(t *testing.T) {
 		ftc.vrdbClient.ExpectRequest(`update _vt.vreplication set message='Picked source tablet: cell:\"zone1\" uid:200' where id=1`, &sqltypes.Result{}, nil)
 		ftc.vrdbClient.ExpectRequest(setSessionTZ, &sqltypes.Result{}, nil)
 		ftc.vrdbClient.ExpectRequest(setNames, &sqltypes.Result{}, nil)
-		ftc.vrdbClient.ExpectRequest(setSQLMode, &sqltypes.Result{}, nil)
-		ftc.vrdbClient.ExpectRequest(getSQLMode, sqltypes.MakeTestResult(
```

Comment:
don't see this in the ignored map above?

Reply:
What exactly are you not seeing and where? https://github.com/vitessio/vitess/pull/13765/files#diff-250405c38cc7363ec43eb8828d16b8bd1ff2ccdf29ccbec01202782f4d691d66R61-R82